packages/metadata/README.md

```diff
@@ -0,0 +1,27 @@
+# Metadata
+
+Package containing metadata service
+
+## Installation
+Npm
+`npm install --save @sap-ux/metadata`
+
+Yarn
+`yarn add @sap-ux/metadata`
+
+Pnpm
+`pnpm add @sap-ux/metadata`
+
+## Usage
+
+1. Import the needed functions in your modules
+
+    ```typescript
+    import { MetadataService  } from '@sap/ux-metadata';
+    
+    const metadata = new MetadataService({ ODataVersion: '4.0' });
+    metadata.import((v4metadata as any).metadata, 'testDummy/metadataV4.xml');
```

Comment:
It is not clear what variable `v4metadata` is coming and what it represents. Also we should not use type assertions in examples.

Reply:
created element. done